src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInMdx.ts

```diff
@@ -20,28 +20,28 @@ export const migrateCalculatedMeasuresInMdx = ({
   mdx,
   serverKey,
   dataModels,
-  namesOfCalculatedMeasurestoMigrate,
+  namesOfCalculatedMeasuresToMigrate,
 }: {
   mdx: MdxSelect | MdxDrillthrough;
   serverKey?: string;
   dataModels: { [serverKey: string]: DataModel };
-  namesOfCalculatedMeasurestoMigrate: string[];
+  namesOfCalculatedMeasuresToMigrate: string[];
 }): {
   cubeName: CubeName;
-  namesOfCalculatedMeasuresToMigrateInWidget: string[];
+  namesOfCalculatedMeasuresRemovedFromMdx: string[];
```

Comment:
Answering [this comment](https://github.com/activeviam/activeui-migration/pull/70#discussion_r1061759629).

I'm happy to keep `migrateCalculatedMeasuresInMdx`, but then I'd change `namesOfCalculatedMeasuresRemovedFromMdx` to be consistent. What about `namesOfMigratedCalulatedMeasures`?

Reply:
Done ✅